README.md

```diff
@@ -16,6 +16,23 @@ you might also like
 
 [Upcase]: https://upcase.com?utm_source=til
 
+## Contributing a TIL
+
+TIL's should be a focused and concise piece of knowledge that's interesting and
```

Comment:
Do you think if it makes sense to add that TILs should be practically applicable rather than just worth sharing?